frontend/src/__tests__/cypress/cypress/e2e/modelRegistry/ModelRegistry.cy.ts

```diff
@@ -35,4 +39,32 @@ describe('Model Registry Global', () => {
 
     modelRegistry.tabEnabled();
   });
+  it('No registered models in the selected Model Registry', () => {
+    initIntercepts({
+      disableModelRegistryFeature: false,
+    });
+
+    cy.intercept(
+      '/api/k8s/apis/modelregistry.opendatahub.io/v1alpha1/modelregistries',
+      mockModelRegistries,
+    );
+    cy.intercept(
+      '/api/k8s/apis/modelregistry.opendatahub.io/v1alpha1/namespaces/shared/modelregistries/modelregistry-sample',
+      mockSingleModelRegistry,
+    );
+    cy.intercept(
+      '/api/k8s/apis/route.openshift.io/v1/namespaces/shared/routes/modelregistry-sample',
+      mockSingleModelRegistryRoute,
+    );
+
+    cy.intercept(
+      { method: 'POST', pathname: '/api/proxy/api/model_registry/v1alpha2/registered_models' },
+      mockRegisteredModelList({ size: 0 }),
+    );
```

Comment:
These will be usefull for other test cases also, you might move these intercepts to `initIntercepts` and pass a variable for `size: 0` to mock for empty state.